src/scripts/directives.js

```diff
@@ -66,7 +66,9 @@
           link: function(scope, element, attrs, ctrl, transclude) {
             if (scope.message.compileContent) {
               var transcludedEl;
-
+              
+              var compileScope = scope.message.compileToScope || scope.$parent;
```

Comment:
no need to have a variable here since it is used only once, you can move it just to the place